src/Juvix/Core/Pipeline.hs

```diff
@@ -15,27 +14,19 @@ import qualified Juvix.Core.Translate as Translate
 import qualified Juvix.Core.Types as Types
 import Juvix.Library
 import qualified Juvix.Library.Usage as Usage
-import qualified Michelson.TypeCheck as Michelson
-import qualified Michelson.Untyped as Michelson
 
-type RawMichelson f = f Michelson.PrimTy Michelson.RawPrimVal
+type RawTerm ty val = IR.Term ty val 
 
-type RawMichelsonTerm = RawMichelson IR.Term
+type RawElim ty val = IR.Elim ty val 
 
-type RawMichelsonElim = RawMichelson IR.Elim
+type Term ty val = HR.Term ty val
```

Comment:
Just a general remark: do we want to use these kind of aliases? Outside of the module it is not clear that for example a `RawTerm` is equal to a `IR.Term`. I think there are some other places where similar aliases are used, and unless we expect the aliases to change, I don't see a reason to use them.

Reply:
I agree